pkg/operator/sync.go

```diff
@@ -656,6 +662,11 @@ func newTerminationDaemonSet(config *OperatorConfig) *appsv1.DaemonSet {
 			},
 		},
 		Spec: appsv1.DaemonSetSpec{
+			UpdateStrategy: appsv1.DaemonSetUpdateStrategy{
+				RollingUpdate: &appsv1.RollingUpdateDaemonSet{
+					MaxUnavailable: &daemonsetMaxUnavailable,
```

Comment:
i guess this needs to be a `*intstr.IntOrString` or cast to one

Reply:
When it is declared you can call `intstr.MustParse` IIRC to get it to be in the right format (it's a struct so best to use their parse util)